content/en/blog/_posts/Introducing-Kueue.md

```diff
@@ -0,0 +1,197 @@
+
+For each namespace, we define a LocalQueue that points to the ClusterQueue above.
+
+```yaml
+apiVersion: kueue.x-k8s.io/v1alpha2
+kind: LocalQueue
+metadata:
+  name: training
+  namespace: team-ml
+spec:
+  clusterQueue: research-pool
+```
+
+Admins create the above setup once. As batch users, we are able to find the queues we are allowed to submit to by listing the `LocalQueues` 
+in our namespace. The command is similar to the following: `kubectl get -n my-namespace localqueues`
+
+To submit a job, we simply set the `kueue.x-k8s.io/queue-name` annotation as follows:
+
+```yaml
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: sample-job-
+  annotations:
+    kueue.x-k8s.io/queue-name: main
+spec:
+  parallelism: 3
+  completions: 3
+  template:
+    spec:
+      tolerations:
+      - key: spot
+        operator: "Exists"
+        effect: "NoSchedule"
+      containers:
+      - name: dummy-job
+        image: gcr.io/k8s-staging-perf-tests/sleep:latest
+        args: ["30s"]
+        resources:
+          requests:
+            cpu: 1
+      restartPolicy: Never
+```
+
+
+Kueue operates by forcing the job to be created in suspended mode. Once the job is at the head of the ClusterQueue, Kueue evaluates if it can start
+by checking if the resources requested by the job fit the available quota. 
+
+In the above example, the job tolerates Spot resources, and so if there are previously admitted jobs consuming all existing on-demand quota but
+not all of Spot’s, Kueue admits the job using the Spot quota. Kueue does this by issuing a single update to the job object that: 
+- Changes the suspended flag to false 
+- Adds the nodeSelector term instance-type: spot to the pod template so that when the pods are created by the job controller, they can only schedule 
+  on Spot nodes. 
+
+Finally, if the nodes are not actually provisioned yet, then kube-scheduler will initially mark the pods as unschedulable, which will trigger the 
+cluster-autoscaler to provision the nodes.
+
+## Future work and getting involved
+The example above offers a glimpse of some of Kueue’s features including support for quota, resource flexibility and integration with cluster 
+autoscaler. Kueue also supports fair-sharing, job priorities and different queueing strategies. Take a look at 
+[Kueue documentation](https://github.com/kubernetes-sigs/kueue/tree/main/docs) to learn more about those features and how to use Kueue. 
+
+We have a number of features that we plan to add to Kueue, such as hierarchical quota, budgets, and support for dynamically sized jobs. In 
+the more immediate future, we are focused on adding support for job preemption.
+
+[The latest Kueue release is available on Github](https://github.com/kubernetes-sigs/kueue/releases), try it out if you run batch workloads. 
+We are in the early stages of this project and we are seeking feedback of all levels, major or minor, so please don’t hesitate to reach out. We’re 
```

Comment:
Let's give a shout out to our contributors :)

Reply:
good call, added.